doc/conf.py

```diff
@@ -47,7 +47,7 @@
 def setup(app):
     # from sphinx.ext.autodoc import cut_lines
     # app.connect('autodoc-process-docstring', cut_lines(4, what=['module']))
-    app.add_description_unit(
+    app.add_object_type(
```

Comment:
please explain this change :+1:

Reply:
See https://github.com/tox-dev/tox/pull/841/commits/116afb14d12fc16e2f7a98b685bd7c7bab2e125c